docs/architecture/tendermint-core/adr-001-logging.md

```diff
@@ -26,6 +26,7 @@ First, we will need an interface for all of our libraries (`tmlibs`, Tendermint,
 type Logger interface {
     Debug(msg string, keyvals ...interface{}) error
     Info(msg string, keyvals ...interface{}) error
+    Warn(msg string, keyvals ...interface{}) error
```

Comment:
Taken AI to upstream this

Reply:
Probably not worth updating such an old ADR (ADR tend to be historical, rather then reflecting the latest status). I would revert all changes in this file, and find where to insert them in the documentation.

Reply:
Resolved